Code.gs

```diff
@@ -161,6 +174,8 @@ function main(){
           
         if (emailWhenAdded)
           GmailApp.sendEmail(email, "New Event Added", "New event added to calendar \"" + targetCalendarName + "\" at " + event.startTime);
+          
+        Utilities.sleep(250); //Solve some "Creating too many events in a short time" issues
```

Comment:
This seems unrelated to handling all day events, isn't it?

Reply:
Sure. Do you think this should be separate, then?

Reply:
Well yes probably :-)

Reply:
Ok, this should be fixed now